client/src/components/RelatedItems/ActionButton.jsx

```diff
@@ -4,15 +4,22 @@ import starButton from './images/starButton.png';
 import deleteButton from './images/deleteButton.png';
 
 
-function ActionButton({productCardSetType}) {
-  let buttonImageSrc = productCardSetType === 'RelatedProducts' ? starButton : deleteButton;
-  return (
-    <input type="image" src={buttonImageSrc} data-bs-toggle="modal" data-bs-target="#comparisonModal" style={{width: 'auto', height: '5%', position: 'absolute', top: '10px', right: '10px', backgroundColor: 'lightGrey', borderRadius: '50%'}}></input>
-  );
+function ActionButton({productCardType, removeProductFromFavorites, productId}) {
+  if (productCardType === 'YourOutfit') {
+    return (
+      <input type="image" src={deleteButton} onClick={() => {removeProductFromFavorites(productId);}} style={{width: 'auto', height: '5%', position: 'absolute', top: '10px', right: '10px', backgroundColor: 'lightGrey', borderRadius: '50%'}}></input>
+    );
+  } else {
+    return (
+      <input type="image" src={starButton} data-bs-toggle="modal" data-bs-target="#comparisonModal" style={{width: 'auto', height: '5%', position: 'absolute', top: '10px', right: '10px', backgroundColor: 'lightGrey', borderRadius: '50%'}}></input>
+    );
+  }
```

Comment:
You could create a class in CSS for some of this styling. It would reduce repeated code!

`style={{width: 'auto', height: '5%', position: 'absolute', top: '10px', right: '10px', backgroundColor: 'lightGrey', borderRadius: '50%'}}`

Reply:
That's my next move actually, put all my inline style into a css file.